book/visual-effects.md

```diff
@@ -650,11 +649,10 @@ example](https://browser.engineering/examples/example11-rounded-background.html)
 www/examples/example11-rounded-background.html
 :::
 
-will round the corners of its background
-(see Figure 1; notice that it does not round the text, though).
+will round the corners of its background (see Figure 1).
```

Comment:
The image changed and the old caption and description (here and also one place way later in the chapter) are no longer referring to anything.